pkg/virt-launcher/virtwrap/converter/compute/graphics.go

```diff
@@ -71,6 +71,18 @@ func (g GraphicsDomainConfigurator) configureVNCSocket(vmi *v1.VirtualMachineIns
 }
 
 func (g GraphicsDomainConfigurator) configureVideoDevice(vmi *v1.VirtualMachineInstance, domain *api.Domain) {
+	if vmi.Spec.Domain.Devices.Video != nil {
```

Comment:
In the original code, this logic came after the architecture dependent logic.
Please keep the original order.